js/reference.js

```diff
@@ -16,6 +16,17 @@
 
 var ERMrest = (function(module) {
 
+    module._contexts = Object.freeze({
```

Comment:
In addition we should create a separate `context.js`, which would contain all these declarations, instead of searching them in different files.